src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,90 @@
 package racingcar;
 
+import camp.nextstep.edu.missionutils.Console;
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
 public class Application {
     public static void main(String[] args) {
         // TODO: 프로그램 구현
+
+        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
+        String carNames = Console.readLine();
+        System.out.println("시도할 회수는 몇회인가요?");
+        //사용자는 몇 번의 이동을 할 것인지를 입력할 수 있어야 한다.
+        int tryNum = Integer.parseInt(Console.readLine());
+
+        //자동차 이름은 쉼표(,)를 기준으로 구분하며 이름은 5자 이하만 가능하다.
+        String carSplit[] = carNames.split(",");
+        Car cars[] = new Car[carSplit.length];
+
+        int idx = 0;
+        for(String carName : carSplit) {
+            if(carName.length() > 5) throw new IllegalArgumentException();
```

Comment:
예외만 던지고 프로그램이 종료가 된다면, 어떤 이유로 프로그램이 종료되었는지 사용자는 알 수 있을까요?
예외처리를 하기 전 사용자에게 어떤 메세지를 남기면 좋을 지 한 번 생각해보면 좋겠어요!
```suggestion
            if(carName.length() > 5) throw new IllegalArgumentException("차 이름은 다섯 글자 이하여야 합니다");
```
처럼 사용한다면 메세지를 전달할 수 있답니다